examples/suggest-psp/README.md

```diff
@@ -0,0 +1,34 @@
+# Suggest Pod Security Policy
```

Comment:
title is misleading, it's suggesting changes not suggesting a whole resource.

Reply:
Renamed